flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/CassandraSinkBase.java

```diff
@@ -140,10 +146,15 @@ protected Session createSession() {
 	public abstract ListenableFuture<V> send(IN value);
 
 	private void tryAcquire() throws InterruptedException, TimeoutException {
```

Comment:
As this function is private, do we still need to keep it? or just call `tryAcquire(int permits)` directly in `invoke`

Reply:
Guess it is a matter of style; I removed the overloaded function, and just call `tryAcquire(1)`